includes/widgets/alert.php

```diff
@@ -147,6 +148,41 @@ protected function register_controls() {
 			]
 		);
 
+		$this->add_control(
+			'selected_icon',
```

Comment:
Please use semantic controls names. 
```suggestion
			'dismiss_icon',
```

Reply:
@matipojo Done